src/com/chess/ChessPlayerFactory.java

```diff
@@ -50,7 +50,7 @@
                     players[i] = new MinMaxAIChessPlayer(colors[i], 1);
                     break;
                 case "H":
-                    players[i] = new MinMaxAIChessPlayer(colors[i], 4);
+                    players[i] = new MinMaxAIChessPlayer(colors[i], 2);

```

Comment:
Had to lower this since chess has a lot bigger state tree then checkers. We could probably make it more efficient so it can go deeper later